webpack.config.js

```diff
@@ -173,6 +175,73 @@ const standardConfig = merge(commonConfig, {
   target: 'web'
 })
 
+const fuzzyConfig = merge(commonConfig, {
```

Comment:
Using webpack to create these fuzzy files instead of copying the explicitly versioned files to the additional filename formats has a couple possible downsides:
- It's slightly slower (no big deal but it's a little something).
- The output is not exactly the same for the fuzzies as for the primary (maybe no big deal, but it makes it harder to validate that it's actually the same build). E.g., in my test I found that `installedChunks` variable is different in the outputs of the primary config vs the outputs of the fuzzyConfig, making the latter files a few bytes larger than their non-fuzzy counterparts.
- It adds another configuration to maintain.

Reply:
@bjfield this has been updated to a file-clone script, would you mind taking another look?